tests/cli/test_compose_tar_kickstart.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+# Note: execute this file from the project root directory
+
+#####
+#
+# Build tar image and install it using liveimg kickstart command
+#
+#####
+
+set -e
+
+. /usr/share/beakerlib/beakerlib.sh
+. $(dirname $0)/lib/lib.sh
+
+CLI="${CLI:-./src/bin/composer-cli}"
+QEMU="qemu-system-$(uname -m)"
+QEMU_CMD="$QEMU -machine accel=kvm:tcg"
```

Comment:
These should not be needed, right? Why not use QEMU which is defined in the shared library ?

Reply:
Not exactly in accordance to the variables from Brian's commit, but I changed the variables to $QEMU_BIN (path to the qemu-system binary) and $QEMU (qemu-system binary with the accelerator parameters), and therefore there is no need to define them anymore in the test script.